docs/blog/unettest.md

```diff
@@ -0,0 +1,291 @@
+# Network Rules as (Tested) Code
+
+![What is the answer, O Deep Thought](https://docs-unettest.s3.us-east-2.amazonaws.com/o_deep_thought.png)
+
+When we write software, there are many ways to ask a question. We could go to
+Stack Overflow or to a smarter coworker, but I like to ask my computer first.
+There are many ways to ask computers questions, and one of the most effective is
+with TDD. TDD is awesome. We can ask many questions quickly, and once a question
+is asked, we can be confident it will remain answered. Given a question on how
+something should work, write a test to describe the behavior. Then run the tests
+and write code to make the test green. Great stuff. If there is a corner case, a
+test case can expose it. When code turns red, make it better.
+
+I was writing NGINX configurations some time ago. I was new to NGINX and trying
+to figure it out. Their documentation (lovely though it is) can be impenetrable.
+I was grokking the config language by looking at existing configurations and
+mimicking what they do. If I had a question, I could not ask it of my computer
```

Comment:
because I'm asking you to scrap the first paragraph, you'll have to rework your sentences about having questions since the context is now gone.